src/setup/devnet/index.js

```diff
@@ -404,13 +404,58 @@ export class Devnet {
+                        ], {stdio: getRemoteStdio()})
+
+                        if (i == 0) {
+                            await execa('ssh', [
+                                `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
+                                `-i`, `~/cert.pem`,
+                                `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                                `bash ${this.config.targetDirectory}/service-host.sh`
+                            ], {stdio: getRemoteStdio()})
+
+                            // TODO: Target location would vary depending on bor/heimdall version
+                            await execa('ssh', [
+                                `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
+                                `-i`, `~/cert.pem`,
+                                `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                                `sudo mv /home/ubuntu/ganache.service /lib/systemd/system/`
+                            ], {stdio: getRemoteStdio()})
+ 
+                        }
+                        else {
+                            await execa('ssh', [
+                                `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
+                                `-i`, `~/cert.pem`,
+                                `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                                `bash /home/${this.config.devnetBorUsers[i]}/service.sh`
+                            ], {stdio: getRemoteStdio()})
+                        }
+    
+                        // TODO: Target location would vary depending on bor/heimdall version
+                        await execa('ssh', [
+                            `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
+                            `-i`, `~/cert.pem`,
+                            `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                            `sudo mv /home/ubuntu/bor.service /lib/systemd/system/`
+                        ], {stdio: getRemoteStdio()})
+                                                
+                        // TODO: Target location would vary depending on bor/heimdall version
+                        await execa('ssh', [
                             `-o`, `StrictHostKeyChecking=no`, `-o`, `UserKnownHostsFile=/dev/null`,
                             `-i`, `~/cert.pem`,
-                            `${ganacheUser}@${ganacheURL.hostname}`,
-                            `tmux new -d -s matic-cli-ganache; tmux send-keys -t matic-cli-ganache:0 'bash ~/ganache-start-remote.sh' ENTER`],
-                        {stdio: getRemoteStdio()})
+                            `${this.config.devnetBorUsers[i]}@${this.config.devnetBorHosts[i]}`,
+                            `sudo mv /home/ubuntu/heimdalld.service /lib/systemd/system/`
```

Comment:
can we use `~` instead of `/home/ubuntu` ? 
This would work with any created username

Reply:
Same everywhere

Reply:
Fixed: [2fcdeef](https://github.com/maticnetwork/matic-cli/pull/91/commits/2fcdeefe0e7bf777827b66c6af5a756f8d4f4504)